sbin/pfdns

```diff
@@ -214,8 +220,70 @@ sub response_handler {
            return  regzone($qname, $qclass, $qtype, $peerhost,$query,$conn);
         }
     }
+    
+    foreach my $network (@routed_dns_enforcement_nets_named) {
+        if ($network->contains($ip)) {
+            $logger->trace("Performing DNS enforcement on $ip");
+            my $mac = pf::iplog::ip2mac($ip_addr);
+
+            unless($mac){
+                $logger->warn("Can't lookup IP $ip_addr to a MAC address. Returning result as if unreg.");
+                return regzone($qname, $qclass, $qtype, $peerhost,$query,$conn);
+            }
+
+            my $node = node_attributes($mac);
+            if($node->{status} eq $STATUS_REGISTERED){
+                $logger->info("Node $mac is registered, returning resolved domain.");
```

Comment:
A detail but since we have a prefix in the log with the mac address do we still have to use the $mac ?